docs/source/flow-group.rst

```diff
@@ -0,0 +1,109 @@
+.. _flow-group:
+
+==========
+FlowGroups
+==========
+
+This chapter provides information about submitting and running groups of
+multiple operations with varying directives.
+
+.. _flow_group_definition:
+
+Definition
+==========
+
+A :py:class:`FlowGroup` is a collection of one or more operation(s) with the
+directives associated with that operation. Operations can be in multiple groups
+and the directives for an operation can be different in every group that it is
+in. Practically, a :py:class:`FlowGroup` acts like a "meta-operation" that can
+be submitted and run identical to operations. In fact, within the source code,
+all operations are wrapped by singleton groups that handle command generation
+and resource requests for that operation.
+
+.. _flow_group_basic_usage:
+
+Basic Usage
+===========
+
+While a :py:class:`FlowGroup` is automatically created for each operation, users must
+manually create multiple operation groups. Below is an example which creates a
+group named ``ex`` which contains the operations ``op1`` and ``op2``.
+
+.. code-block:: python
+
+    # project.py
+    from flow import FlowProject
+
+    class Project(FlowProject):
+        pass
+
+    ex = Project.make_group(name='ex')
+
+    @ex
+    @Project.operation
+    def op1(job):
+        pass
+
+    @ex
+    @Project.operation
+    def op2(job):
+        pass
+
+    if __name__ == '__main__':
+        Project().main()
+
+To execute or submit ``ex`` alone, use the ``-o`` option to select the group like you would for a
+regular operation. Groups are eligible if at least one of their operations
+is eligible. Resources are requested appropriately whether groups are run in
+serial or parallel.
+
+.. tip::
+
+    To avoid large resource requests, avoid running multiple operation groups in
+    parallel. Also, when submitting multiple operation groups in serial, remember
+    that the resources requested will match the most computationally intensive
+    operation in that group. To avoid wasting compute time, make sure that you group operations
+    with similar resource requests or that cheaper operations do not run for long.
```

Comment:
I think this tip has two tips: one against running in parallel; and one about how to allocate operations into groups. Should it be split?